features/step_definitions/networking.rb

```diff
@@ -639,7 +639,7 @@
 
   desired_multus_replicas = daemon_set('multus', project('openshift-multus')).replica_counters(user: admin)[:desired]
   available_multus_replicas = daemon_set('multus', project('openshift-multus')).replica_counters(user: admin)[:available]
-
+  cb.desired_multus_replicas = desired_multus_replicas
```

Comment:
@anuragthehatter perhaps leave a comment in the file indication `desired_multus_replicas` is saved into a clipboard

Reply:
Yep, that would be great. Thanks. Done.